### ISSUE TITLE
Migrate Web Analytics prototype to PDS Cloud

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/cloud-tasks#29]
## 💡 Description

Migrate the deployed https://github.com/NASA-PDS/web-analytics, Athena, and Quicksight instances from @kaipak's group's AWS to PDS

Sub-tasks to be identified by @jimmie

**Comment by jordanpadams** [comment]
📆 May status: In progress. Delayed 1 sprint. No impact anticipated.

**Comment by jordanpadams** [comment]
📆 July status: delayed on additional sprint. This is a non-critical task and can be completed anytime during this or next build.

**Comment by jordanpadams** [comment]
📆 August status: expected to be done by end of sprint, 9/8

**Comment by jimmie** [comment]
Should have this stood up to the point of being able to query from Athena but am waiting on permissions (as usual). Further discussion will be necessary regarding Quicksight since it requires additional AWS subscriptions.

**Comment by jordanpadams** [comment]
:calendar: Sept Status: No impact on build. Moving to B13.1 work effort

**Comment by jordanpadams** [comment]
:calendar: December status: Pushed out 1 more sprint. no affect on B13.1 delivery.

**Comment by jordanpadams** [comment]
📆 January status: Delayed several sprints to deal with registry issues. No impact on delivery. Not required for B13.1

**Comment by jordanpadams** [comment]
:calendar: March status: Deferred to B14.0 due to some SA acces issues. No impact on delivery. This is in work and part of "beta" efforts

**Comment by jordanpadams** [comment]
📆 April status: Migration and improvement of dashboards ongoing. Scope increase to include data sync scripts. Beta task. No impact on build.

**Comment by jordanpadams** [comment]
📆 May status: Migration completed. Working on new infra, cron jobs, and improved reporting. New tasks to come
